packages/jsii-rosetta/lib/fixtures.ts

```diff
@@ -46,10 +49,22 @@ export function fixturize(snippet: TypeScriptSnippet): TypeScriptSnippet {
   };
 }
 
-function loadLiterateSource(directory: string, literateFileName: string) {
+function loadLiterateSource(
+  directory: string,
+  literateFileName: string,
+  loose = false,
+) {
   const fullPath = path.join(directory, literateFileName);
   const exists = fs.existsSync(fullPath);
   if (!exists) {
+    if (loose) {
+      // In loose mode, we'll fall back to the `.js` file if it exists...
+      const jsFile = fullPath.replace(/\.ts(x?)$/, '.js$1');
+      if (fs.existsSync(jsFile)) {
+        return fs.readFileSync(jsFile, { encoding: 'utf-8' });
```

Comment:
Maybe add a header here saying we are showing `js` code because the expected file is missing?

Reply:
Oh well, we tried